IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -316,6 +316,7 @@ - (BOOL)matchesWithRealm:(nullable NSString *)realm
         {
             return YES;
         }
+        else return NO;
```

Comment:
nit: no need for else here right?

Reply:
there's a need, because otherwise it'd return YES in the end even if neither of clientID nor familyID matches.